rules/svt.rules

```diff
@@ -41,15 +44,14 @@ when
 	Member of Heating_Mode received update
 	or
 	Member of Heating_CalcTime received update
-	or
-	System started
+	//or
+	//System started
 
 then
-	val DELTA=0.2
-	val DATABASE="rrd4j" //"influxdb"
-	val room =  //which room requires heating. If no item triggered  the rule (system started), check for nulls
-		if (triggeringItem === null){ //system started trigger
-			logInfo("svt2","System started, resetting timers and checking for NULL")
+	val double DELTA = 0.2
```

Comment:
This can be removed. I just tried figuring out what causes the warning we discussed in the forum. However I have found it and this was not there.
However we need to specify explicitly the types in some situation. I will mark them here.